core/storage/user/gae_models_test.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+from core.platform import models
+from core.tests import test_utils
+
+(user_models,) = models.Registry.import_models([models.NAMES.user])
+
+
+class ExplorationUserDataModelTest(test_utils.GenericTestBase):
+    """Tests for the ExplorationUserDataModel class."""
+
+    DATETIME_OBJECT = datetime.datetime.strptime('2016-02-16', '%Y-%m-%d')
+    USER_ID = 'user_id'
+    EXP_ID_ONE = 'exp_id_one'
+    EXP_ID_TWO = 'exp_id_two'
+
+    def setUp(self):
+        super(ExplorationUserDataModelTest, self).setUp()
+        user_models.ExplorationUserDataModel(
+            id='%s.%s' % (self.USER_ID, self.EXP_ID_ONE), user_id=self.USER_ID,
+            exploration_id=self.EXP_ID_ONE, rating=2,
+            rated_on=self.DATETIME_OBJECT,
+            draft_change_list={'new_content': {}},
+            draft_change_list_last_updated=self.DATETIME_OBJECT,
+            draft_change_list_exp_version=3).put()
+
+    def test_create_success(self):
+        user_models.ExplorationUserDataModel.create(
+            self.USER_ID, self.EXP_ID_TWO).put()
+        expected_object = user_models.ExplorationUserDataModel(
+            user_id=self.USER_ID, exploration_id=self.EXP_ID_TWO)
+        actual_object = user_models.ExplorationUserDataModel.get_by_id(
+            '%s.%s' % (self.USER_ID, self.EXP_ID_TWO))
+
+        self.assertEqual(
+            expected_object.exploration_id, actual_object.exploration_id)
```

Comment:
Better to do self.assertEqual(self.EXP_ID_TWO, actual_object.exploration_id) -- just compare the properties. Ditto below.